main.nf

```diff
@@ -433,8 +433,14 @@ workflow NFCORE_PROTEINFOLD {
         PREPARE_BOLTZ_DBS(
             params.boltz_ccd_path,
             params.boltz_model_path,
+            params.boltz2_aff_path,
+            params.boltz2_conf_path,
```

Comment:
I also think some of these parameters have not been added to the `Parameters` table of the changelog